logic/logic.go

```diff
@@ -29,16 +32,31 @@ type operation struct {
 	Group         []string
 }
 
+// Deprecated
```

Comment:
@ahangsu Minor:  Based on my read of https://github.com/golang/go/wiki/Deprecated, we ought to tweak deprecation format here.

> To signal that an identifier should not be used, add a paragraph to its doc comment that begins with `Deprecated:` followed by some information about the deprecation, and a recommendation on what to use instead, if applicable.

Reply:
I see, that makes sense, we can extend the deprecation note from java one to the golang `// Deprecated: ...`.